docs/content/en/functions/hmac.md

```diff
@@ -0,0 +1,35 @@
+---
+title: hmac
+# linktitle: hmac
+description: Compute the cryptographic checksum of a message.
+godocref:
+date: 2020-05-29
+publishdate: 2020-05-29
+lastmod: 2020-05-29
+categories: [functions]
+menu:
+  docs:
+    parent: "functions"
+keywords: [hmac,checksum]
+signature: ["hmac HASH_TYPE MESSAGE KEY"]
+workson: []
+hugoversion:
+relatedfuncs: [hmac]
+deprecated: false
+aliases: [hmac]
+---
+
+`hmac` hashes the concatenation of a message and a secret key with the given hash function
```

Comment:
I don't really like this description since HMAC is not simply the hash of message+key.  I'd suggest:
`hmac returns a cryptographic hash that uses a key to sign a message.`